src/deluge/model/output.h

```diff
@@ -154,6 +154,7 @@ class Output {
 	virtual ModelStackWithAutoParam* getModelStackWithParam(ModelStackWithTimelineCounter* modelStack, Clip* clip,
 	                                                        int32_t paramID,
 	                                                        deluge::modulation::params::Kind paramKind) = 0;
+	virtual bool needsPriority() { return false; }
```

Comment:
```suggestion
    /// When true, this clip must be ticked in the early pass, as it affects other clips in the song.
	virtual bool needsEarlyPlayback() const { return false; }
```

`needsEarlyPlayback` might be a better name, and should be marked `const`.

Reply:
Made that change, and now found an issue where audition notes get stuck. Converted to draft until I fix that